apps/analytics/serializers.py

```diff
@@ -0,0 +1,10 @@
+from rest_framework import serializers
+
+
+class SubmissionStats(object):
```

Comment:
I would prefer to keep this in the `jobs` app where the model for this serializer resides. :) Please let me know what you think about this.